pkg/sentry/socket/hostinet/socket_unsafe.go

```diff
@@ -90,26 +90,50 @@ func getsockopt(fd int, level, name int, optlen int) ([]byte, error) {
 	return opt[:optlen32], nil
 }
 
+// GetAddress reads an sockaddr struct from the given address and converts
+// it to the format of SockAddrInet or SockAddrInet6 or raw bytes depending
+// on family.
+func GetAddress(family int, addr []byte) (interface{}, uint32, *syserr.Error) {
```

Comment:
GetAddress doesn't need to be exported does it? -> getAddress

Reply:
Actually, we may need this in other network implementations. So I prefer to export it.